post_processing/post_processing_funcs.jl

```diff
@@ -130,6 +130,13 @@ function postprocessing_box(sol, output_dir)
         clim = (-0.1, 0.1),
     )
     Plots.png(p, joinpath(output_dir, "w.png"))
+    qt = Y.c.ρq_tot ./ Y.c.ρ
```

Comment:
This will cause a problem for the dry case. You can add a check for whether `ρq_tot` is in `Y.c`.

Reply:
Right! thanks